### DISCUSSION TITLE
据说下单前会有额外的备注信息需要填写

### DISCUSSION THREAD

**Discussion by Ghostori** [mikumifa/cppTickerBuy#19] [提问专区]
下图是小芒的界面。想问一下如果CPP这边也需要填写是否为COSER入场的话，大佬会不会出手解决这个问题？
（亦或说可以自行在配置文件中添加有关这个的字段？）
![Screenshot_2024-09-26-18-27-23-08_a2db1b9502c98f25523e43284b79cce6](https://github.com/user-attachments/assets/fe91b32d-7155-4a1b-8ec9-aac257c14339)

**Comment by mikumifa** [comment]
好解决的，不用担心
